spec/dataframe_spec.rb

```diff
@@ -501,7 +501,7 @@
     end
   end
 
-  context "#[]=" do
+  context "[]=" do
```

Comment:
why remove the `#`? It has a meaning, that it's an instance method being tested.